### PR TITLE
Update free-programming-books-my.md

### PR DESCRIPTION
**Pull request by NayLin99**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-my.md [hunk 1]
```diff
@@ -30,4 +30,5 @@
 ### Web Development
 
 * [Professional Web Developer](http://eimaung.com/professional-web-developer) - Ei Maung
+* [Professional Web Developer 2022](https://eimaung.com/pwd2022/) - Ei Maung
```
Comment (davorpa) [inline books/free-programming-books-my.md:33]: Complete with the format of file to download

```suggestion
* [Professional Web Developer 2022](https://eimaung.com/pwd2022/) - Ei Maung (PDF)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to apply some changes. When done, request for a review again using GitHub :arrows_clockwise: button. Meanwhile, draft status can be used to work on it if you want.

**Comment by NayLin99** [comment]
I already made a new pull request #6746 which also include Professional Web Developer 2022. So can you help me close this pull request?

**Comment by davorpa** [comment]
Duplicate of #6746

**Pull request closed without merging**